schema/schema.cc

```diff
@@ -1622,6 +1648,22 @@ const ::tombstone_gc_options& schema::tombstone_gc_options() const {
     return default_tombstone_gc_options;
 }
 
+bool schema::has_tablet_options() const noexcept {
+    return _raw._tablet_options.has_value();
+}
+
+db::tablet_options schema::tablet_options() const noexcept {
```

Comment:
This can throw due to copies.

Reply:
Actually, not due to copies, but parsing. tablet_options constructor is not marked noexcept, and can throw `syntax_exception`. Maybe that should never happen in practice, but I think it's unsafe that this getter is marked noexcept but calls non-noexcept code. It means unnecessary abort in the future, and no one really relies on it to be noexcept.

Reply:
right.  I left it by mistake IIRC as originally it returned a reference to the tablet_options member

Reply:
cf_prop_defs::validate is calling tablet_options::validate that should prevent any parsing errors here, but still let's be more permissive here, just in case.